src/common/com/intellij/plugins/haxe/ide/index/HaxeComponentFileNameIndex.java

```diff
@@ -99,17 +101,26 @@ public static boolean getFileNames(@NotNull String qName,
     @NotNull
     public Map<String, Void> map(final FileContent inputData) {
       final PsiFile psiFile = inputData.getPsiFile();
+      final String moduleName = FileUtil.getNameWithoutExtension(inputData.getFileName());
       final List<HaxeClass> classes = HaxeResolveUtil.findComponentDeclarations(psiFile);
       if (classes.isEmpty()) {
         return Collections.emptyMap();
```

Comment:
Actually if we have file `sss.SEmptyModule` with just `package sss;`
- it can be imported: `import sss.SEmptyModule;` is valid and compiles without errors
- but for example `var a:sss.SEmptyModule` - is not valid: haxe will tell you: `Module sss.SEmptyModule does not define type SEmptyModule`

So maybe we should remove checking at new line 106: 
```
if (classes.isEmpty()) {
          return Collections.emptyMap();
```
What do you think about that?